ui/analyse/src/study/relay/relayView.ts

```diff
@@ -60,12 +60,37 @@ export function renderStreamerMenu(relay: RelayCtrl) {
   );
 }
 
+export function renderPinnedImage(relay: RelayCtrl) {
+  if (
+    window.getComputedStyle(document.body).getPropertyValue('--allow-video') !== 'true' ||
```

Comment:
note to self: refactor/cache that in a ctrl.prop and invalidate on window resize